packages/clerk-js/src/core/devBrowser.ts

```diff
@@ -0,0 +1,105 @@
+import { getDevBrowserJWTFromURL, setDevBrowserJWTInURL } from '@clerk/shared/devBrowser';
+
+import { isDevOrStagingUrl } from '../utils';
+import { getDevBrowserCookie, removeDevBrowserCookie, setDevBrowserCookie } from '../utils/cookies/devBrowser';
+import { clerkErrorDevInitFailed } from './errors';
+import type { FapiClient } from './fapiClient';
+
+export interface DevBrowser {
+  clear(): void;
+
+  setup(): Promise<void>;
+
+  getDevBrowserJWT(): string | undefined;
+
+  setDevBrowserJWT(jwt: string): void;
+
+  removeDevBrowserJWT(): void;
+}
+
+export type CreateDevBrowserOptions = {
+  frontendApi: string;
+  fapiClient: FapiClient;
+};
+
+export function createDevBrowser({ frontendApi, fapiClient }: CreateDevBrowserOptions): DevBrowser {
+  function getDevBrowserJWT() {
+    return getDevBrowserCookie();
+  }
+
+  function setDevBrowserJWT(jwt: string) {
+    setDevBrowserCookie(jwt);
+  }
+
+  function removeDevBrowserJWT() {
+    removeDevBrowserCookie();
+  }
+
+  function clear() {
+    removeDevBrowserJWT();
+  }
+
+  async function setup(): Promise<void> {
+    if (!isDevOrStagingUrl(frontendApi)) {
+      return;
+    }
+
+    // 1. Set network interceptors to Pass dev
+    fapiClient.onBeforeRequest(request => {
+      const devBrowserJWT = getDevBrowserJWT();
+      if (devBrowserJWT && request?.url) {
+        request.url = setDevBrowserJWTInURL(request.url, devBrowserJWT, true);
+      }
+    });
+
+    fapiClient.onAfterResponse((_, response) => {
+      const newDevBrowserJWT = response?.headers?.get('Clerk-Db-Jwt');
```

Comment:
(nit) Can we make this a constant that lives with the others?

Reply:
Fixed.